### PR TITLE
URLBae (Independent Publisher)

### PR DESCRIPTION
**Pull request by troystaylor**
brave_NGne1yd5P8
brave_Lllb5vnp7B

---
#### Important: Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

File: independent-publisher-connectors/URLBae/readme.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# URLBae
```
Comment (v-eisljami) [inline independent-publisher-connectors/URLBae/readme.md:1]: Readme.md doesn't have Deployment instructions. This is one of the required sections. Please follow the official template instructions on this [link](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md).

File: independent-publisher-connectors/URLBae/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,3033 @@
...
+              },
+              "X-RateLimit-Reset": {
+                "description": "X-RateLimit-Reset",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Update a pixel",
+        "description": "To update a your pixels code via the API, you can use this endpoint.",
```
Comment (v-eisljami) [inline independent-publisher-connectors/URLBae/apiDefinition.swagger.json:1666]: All descriptions must be free from grammatical errors.

Comment (troystaylor) [reply independent-publisher-connectors/URLBae/apiDefinition.swagger.json]: This has been fixed.

File: independent-publisher-connectors/URLBae/readme.md (hunk 1 above)
Comment (troystaylor) [reply independent-publisher-connectors/URLBae/readme.md:1]: Not required.

**Review by v-eisljami — APPROVED** [review]

**Review by v-eisljami — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@troystaylor Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**